public/css/login.css

```diff
@@ -0,0 +1,85 @@
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 100%;
+  align-self: center;
+}
+
+.header {
+  width: 100%;
+  height: 40px;
```

Comment:
I would advise not using `px` and take a look at `rem` for more responsive site

Reply:
Ok cool i will do it